src/java/org/myhush/gui/ProvingKeyFetcher.java

```diff
@@ -64,42 +64,35 @@ public void fetchIfMissing(final StartupProgressDialog parent) throws IOExceptio
         }
     }
 
-    // BRX-TODO: Automatically move ZcashParams over to HushParams if it exists
+    // BRX-NOTE: We're tied to 'ZcashParams' right now due to the `hushd` treatment of location this:
```

Comment:
FYI, they are actually ZcashParams, not HushParams. If the Hush network has it's own ceremony/params in the future, we can change the name. But for now, this is nice because if you have a Zcash or any Zcash fork already on a server, installing Hush can use the same keys and vice versa, saving a 1GB file download